packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -63,9 +64,30 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
     });
 
   const threads = sortPinned(sortByFeaturedFilter(data || [], featuredFilter));
+  //
+  //Checks if the current page is a discussion page and if the window is small enough to render the mobile menu
+  //Checks both for mobile device and inner window size for desktop responsiveness
 
   useManageDocumentTitle('Discussions');
 
+  const [toggleMobileView, setToggleMobileView] = useState(
+    location.pathname.includes('discussions') && window.innerWidth <= 425
```

Comment:
again, this 425 should be either defined as a global var or better, this logic should be handled by the utils functions I mentioned above

Reply:
Fixed in next commit